### PR TITLE
FRED-800 FIMS-AZ: fix authorities-scope mapping generation on deploym…

### PR DESCRIPTION
**Pull request by fshihames**
…ent env

- clean up nup-roles.yaml based ASTM 1.0
  (see https://github.com/astm-utm/Protocol/blob/master/utm.yaml)

The key file on this repo for this clean up is this utm-apis/fimsauthz-api/nup-roles.yaml which is used to generate the authorities_to_scope_mapping.json during the deployment process.

The key files in fims-authz repo (also FRED-800-update-nup-roles branch) are being cleaned up are below:
modules/fims-authz-sig/config/clients.json
modules/fims-authz-sig/config/authorities_to_scope_mapping.json
(For your quick reference, I attached both here)

The code that process and creates authorities_to_scope_mapping.json is fims-authz-sig/tools/generate_scopes_json/scope_me.js, which is called by fims-authz-sig/scripts/zip_for_deploy.sh during the deployment.

I want to get your review feedback earlier since you have the insights the legacy setup.  Once the contents are correct, testing will be underway accordingly.  

[clients.json.txt](https://github.com/nasa/utm-apis/files/7646009/clients.json.txt)
[authorities_to_scope_mapping.json.txt](https://github.com/nasa/utm-apis/files/7646011/authorities_to_scope_mapping.json.txt
[scope_me.js.txt](https://github.com/nasa/utm-apis/files/7664147/scope_me.js.txt)
)

### CODE REVIEW

File: fimsauthz-api/nup-roles.yaml [hunk 1]
```diff
@@ -6,106 +6,55 @@ UTM_AUTHORITY:
...
-        - utm.constraint_management
         - utm.constraint_processing
         - utm.conformance_monitoring_sa
-        - utm.availability_arbitration
 
-USS_CONSTRAINT_MANAGER:
+CONSTRAINT_MANAGER:
     description: |
-        A USS with this role can publish airspace constraints such as UASVolumeReservations (UVRs).
+        An user with this role can publish airspace constraints.
```
Comment (issmith1) [inline fimsauthz-api/nup-roles.yaml:32]: a user or service (or simply "This role can..."

File: fimsauthz-api/nup-roles.yaml [hunk 2]
```diff
@@ -6,106 +6,55 @@ UTM_AUTHORITY:
...
         assumptions TBD.
 
         According to the ASTM Spec - 1.0
-        - Client may manage (create, edit, and delete) Constraints they own.
+        - Client may manage (create, edit, and delete) constraints.
 
     authority: UTM_AUTHORITY
     examples:
-        - "A Public Safety USS, acting on behalf of the ANSP, adds an ASTM Contrating (aka UVR) to the UTM Network."
+        - "A Public Safety USS, acting on behalf of the ANSP, adds an ASTM Contrating to the UTM Network."
```
Comment (issmith1) [inline fimsauthz-api/nup-roles.yaml:42]: chg Contrating to Constraint

**Review by issmith1 — COMMENTED** [review]
* add to https://github.com/nasa/utm-apis/blob/master/fimsauthz-api/CHANGELOG.md

File: fimsauthz-api/nup-roles.yaml (hunk 1 above)
Comment (fshihames) [reply fimsauthz-api/nup-roles.yaml]: fixed

**Pull request merged**